packages/core/src/tracing/instrumentation/test/calculator.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+/* eslint-disable no-unused-vars */
+const shimmer = require('../../shimmer');
+const { EXIT } = require('../../constants');
+const tracingUtil = require('../../tracingUtil');
+const cls = require('../../cls');
+const iitmHook = require('../../../util/iitmHook');
+
+let active = false;
+
+/* eslint-disable no-console */
+
+exports.activate = function activate() {
+  active = true;
+};
+
+exports.deactivate = function deactivate() {
+  active = false;
+};
+
+exports.init = function init() {
+  iitmHook.onModuleLoad('esm-square-calculator', instrument);
```

Comment:
`wrapper.onModuleLoad('mysql', true | false)`

or

list in the wrapper which triggers iitm for specific modules for now.